src/Config/Schema.php

```diff
@@ -101,9 +101,11 @@ public function getDefaults(string $stage): array
      */
     public function getSchema(): array
```

Comment:
This would make sense to rename to `getVariables`

Reply:
Make sense, done.